content/en/api_catalog/add_entries.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Adding Entries to API Catalog
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
+- link: "/tracing/api_catalog/explore_and_catalog_apis/"
+  tag: "Documentation"
+  text: "Exploring and cataloging endpoints"
+- link: "/tracing/api_catalog/monitor_apis/"
+  tag: "Documentation"
+  text: "Monitoring APIs"
+aliases:
+    - /tracing/api_catalog/get_started
+---
+
+{{< site-region region="gov,ap1" >}}
+<div class="alert alert-warning">API Catalog is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+## Overview
+
+API Catalog uses APM instrumentation for distributed tracing to automatically discover endpoints your Datadog organization. For instrumented services with supported libraries, endpoints are automatically populated into API Catalog.
+
+Register auto-discovered endpoints, or upload an OpenAPI file to benefit from the full value of API Catalog.
+
+## Register auto-discovered endpoints
+
+Register auto-discovered endpoints to groups of APIs to track their usage, set ownership, and set monitoring policies from a centralized location.
+
+To register endpoints:
+
+1. From the **Explorer** page, select the endpoints to register.
+2. Click **Register Endpoints**.
+3. Select the API(s) you want to group the endpoints into.
+4. Click **Apply**.
+
+{{< img src="tracing/api_catalog/api-catalog-register.png" alt="Select endpoints in API Catalog and click Register Endpoints button." style="width:65%;" >}}
+
+After endpoints are registered, Datadog starts collecting a new endpoint metric for better [monitoring capabilities][6].
+
+Auto discovery is not available for some frameworks. Check compatibility status in the **learn More** button in Datadog. If you still cannot find your endpoints, try uploading a definition file containing them. Datadog automatically starts collecting data on those endpoint definitions after they are uploaded.
```

Comment:
@ddOfir - The Learn More button in the app just points back to the API Catalog docs. This seems circular--do we have a better way to present compatibility status to users?

Reply:
you are totally right. We did it because the "learn more" button is going to be changed, and hold a page for checking computability (the page that is currently called 'setup'). @barashkenazy @barweiss im not mistaken here right?

Anyway i assume it should be released in the next couple of days so i guess we can leave it as is